src/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { marked } from "marked"
+
+/**
+ * Mangle email address
+ *
+ * @param options Options for the extension
```

Comment:
Can you remove this param since there are no options?

Reply:
Done. Well spotted

Reply:
Done. Well spotted